multibot.py

```diff
@@ -51,37 +49,42 @@ def adjustmad(self, pairs, mad):
     def payload(self, pairs, mad):
 
         payload = {
-            "name": self.prefix + "_" + self.subprefix + "_" + self.suffix,
+            "name": self.attributes.get("prefix", "3CQSBOT", self.dca_conf) \
+                    + "_" + self.attributes.get("subprefix", "MULTI", self.dca_conf) \
+                    + "_" + self.attributes.get("suffix", "standard", self.dca_conf),
             "account_id": self.account_data["id"],
             "pairs": pairs,
             "max_active_deals": mad,
-            "base_order_volume": self.attributes.get("bo"),
-            "take_profit": self.attributes.get("tp"),
-            "safety_order_volume": self.attributes.get("so"),
-            "martingale_volume_coefficient": self.attributes.get("os"),
-            "martingale_step_coefficient": self.attributes.get("ss"),
-            "max_safety_orders": self.attributes.get("mstc"),
-            "safety_order_step_percentage": self.attributes.get("sos"),
+            "base_order_volume": self.attributes.get("bo", "", self.dca_conf),
+            "take_profit": self.attributes.get("tp", "", self.dca_conf),
+            "safety_order_volume": self.attributes.get("so", "", self.dca_conf),
+            "martingale_volume_coefficient": self.attributes.get("os", "", self.dca_conf),
+            "martingale_step_coefficient": self.attributes.get("ss", "", self.dca_conf),
+            "max_safety_orders": self.attributes.get("mstc", "", self.dca_conf),
+            "safety_order_step_percentage": self.attributes.get("sos", "", self.dca_conf),
             "take_profit_type": "total",
-            "active_safety_orders_count": self.attributes.get("max"),
-            "cooldown": self.attributes.get("cooldown", 0),
+            "active_safety_orders_count": self.attributes.get("max", "", self.dca_conf),
+            "cooldown": self.attributes.get("cooldown", 0, self.dca_conf),
             "strategy_list": self.strategy(),
-            "trailing_enabled": self.attributes.get("trailing", False),
-            "trailing_deviation": self.attributes.get("trailing_deviation", 0.2),
-            "allowed_deals_on_same_pair": self.attributes.get("sdsp"),
-            "min_volume_btc_24h": self.attributes.get("btc_min_vol", 0),
-            "disable_after_deals_count": self.attributes.get("deals_count", 0),
```

Comment:
@itzwolf Did you test that setting? @M1cha3lV said there was an error at 3Commas side so he removed it from the code.